Kickstarter-iOS/DataSources/PledgePaymentMethodsDataSourceTests.swift

```diff
@@ -0,0 +1,31 @@
+@testable import Kickstarter_Framework
+@testable import KsApi
+import Prelude
+import XCTest
+
+final class PledgePaymentMethodsDataSourceTests: XCTestCase {
+  private let dataSource = PledgePaymentMethodsDataSource()
+  private let tableView = UITableView()
+
+  func testLoadValues() {
+    let cellData = [
+      (
+        card: GraphUserCreditCard.amex,
+        isEnabled: true,
+        isSelected: true,
+        projectCountry: "Country 1"
+      ),
+      (
+        card: GraphUserCreditCard.visa,
+        isEnabled: false,
+        isSelected: false,
+        projectCountry: "Country 2"
+      )
+    ]
+
+    self.dataSource.load(cellData)
+
+    XCTAssertEqual(2, self.dataSource.numberOfSections(in: self.tableView))
+    XCTAssertEqual(2, self.dataSource.numberOfItems(in: 0))
```

Comment:
Maybe also an assertion for the `self.dataSource.numberOfItems(in: 1)`?

Reply:
This is actually addressed in the follow-up PR: https://github.com/kickstarter/ios-oss/pull/1156/files#diff-24128a4cb3c0c847303e6056c2ace3ea